### ISSUE TITLE
Cancel "No show features specified"?

### ISSUE THREAD

**Issue by jloewenstein** [EzerIT/BibleOL#65]
Sometimes I would like the students to focus on even just one detail, and any extra shown detail can be a distraction. 
Is there a way to not have any feature shown? (Just have the marked text and the questions)
Or maybe there is an "empty" feature (or it can be added)?

**Comment by ernstboogert** [comment]
Hi @jloewenstein, you need to specify your request a little more. Do you mean that there would be no 'My View' button that will allow the student to see the other information/features?

**Comment by jloewenstein** [comment]
No, I meant that in the exercise creation page, when no feature is selected to be shown, there is an error message: 
"No show features specified"
And no option but to select at least one feature. 
![image](https://github.com/EzerIT/BibleOL/assets/163320105/f5688950-9ed2-4896-961c-3267beef6151)
